pkg/pubsub/adapter/converters/scheduler.go

```diff
@@ -31,26 +30,22 @@ func convertCloudScheduler(ctx context.Context, msg *pubsub.Message) (*cev2.Even
 	event := cev2.NewEvent(cev2.VersionV1)
 	event.SetID(msg.ID)
 	event.SetTime(msg.PublishTime)
-	event.SetType(v1beta1.CloudSchedulerSourceExecute)
+	event.SetType(v1beta1.CloudSchedulerSourceJobExecutedEventType)
+	event.SetDataSchema(v1beta1.CloudSchedulerSourceEventDataSchema)
 
-	// Set the source and subject if it comes as an attribute.
-	// We added this attributes so that we could identify the scheduler.
-	// We should remove them here.
 	jobName, ok := msg.Attributes[v1beta1.CloudSchedulerSourceJobName]
 	if !ok {
 		return nil, errors.New("received event did not have jobName")
 	}
-	schedulerName, ok := msg.Attributes[v1beta1.CloudSchedulerSourceName]
```

Comment:
if this attribute is not used anymore, we should remove it from the reconciler as well... 
https://github.com/google/knative-gcp/blob/master/pkg/reconciler/events/scheduler/scheduler.go#L127